hedera-node/src/main/java/com/hedera/services/txns/contract/ContractCreateTransitionLogic.java

```diff
@@ -184,7 +188,6 @@ public void doStateTransition() {
 					OperationTracer.NO_TRACING,
 					false);
 			if (result.isSuccessful()) {
```

Comment:
We have a use case for exceptional halt reason due to `INVALID_SOLIDITY_ADDRESS` in contract create transition logics as-well. If a contract tries to do anything to  another contract in its constructor, we will interact with non-existing account